### PR TITLE
fix: (platform) fix for menu position in menu button

### PR DESCRIPTION
**Pull request by DeepakSap14**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/3086

#### Please provide a brief summary of this pull request.
this PR fixes issues that were raised during defect hunting.
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by KevinOkamoto — APPROVED** [review]
Looks good.

File: libs/platform/src/lib/components/menu-button/menu-button.component.ts [hunk 1]
```diff
@@ -1,21 +1,12 @@
-import {
-    Component,
-    Input,
-    Output,
-    EventEmitter,
-    ChangeDetectorRef,
-    ChangeDetectionStrategy,
-    ViewEncapsulation
-} from '@angular/core';
+import { Component, Input, Output, EventEmitter, ChangeDetectorRef, ChangeDetectionStrategy } from '@angular/core';
```
Comment (droshev) [inline libs/platform/src/lib/components/menu-button/menu-button.component.ts:1]: do we need to change the formatting?

**Review by droshev — CHANGES_REQUESTED** [review]
Can we avoid changing the file formatting?

File: libs/platform/src/lib/components/menu-button/menu-button.component.ts (hunk 1 above)
Comment (DeepakSap14) [reply libs/platform/src/lib/components/menu-button/menu-button.component.ts:1]: removed unnecessary changes. but some formatting are done from prettier on saving the file. so have to keep those formatting.

**Review by sKudum — APPROVED** [review]
LGTM

**Comment by puru-hk** [comment]
Tested tooltip is showing for icon and truncated text button.

**Comment by DeepakSap14** [comment]
menu position fixed now.
Screenshot 2020-09-11 at 9 14 52 AM

**Review by salarenko — APPROVED** [review]

**Pull request merged**